### DISCUSSION TITLE
How to automatically create hypertable in TimescaleDB connection?

### DISCUSSION THREAD

**Discussion by serteikas** [dbeaver/dbeaver#13338] [Q&A]
Hello

I have a connection to TimescaleDB using DBeaver and I'm using GUI to create schemas and tables. However, when I create a table in DBeaver for timescaleDB, it creates a table as a PostgreSQL table rather than a hyper table of TimescaleDB.

Is there a way to automatically transform a PostgreSQL table to a TimescaleDB hyper table upon creation? I would assume that it would be default behaviour if you are working with the TimescaleDB connection, but apparently, it is not. Or I am missing something?

Thanks.

**Comment by LonwoLonwo** [comment]
Hello @serteikas 
Can you please tell me more about how to create TimescaleDB hyper tables? 

https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/create/#create-a-hypertable
If I understand correctly, you must select a statement like this 
`SELECT create_hypertable('conditions', 'time');`
for your tables, to turn them into TimescaleDB hyper tables?

**Reply by serteikas** [reply]
Correct. that is the procedure as given in the documentation. Virtually all tables that are used in TimescaleDB are hyper tables. So any table that I create in TimescaleDB connection should be automatically a hyper table (imo)

**Reply by LonwoLonwo** [reply]
Ok. You can create a feature request for this here: https://github.com/dbeaver/dbeaver/issues
But we don't have a test environment now, therefore, unfortunately, we will not get this task quickly.
